docs/tables/aws_s3tables_table_bucket.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Steampipe Table: aws_s3tables_table_bucket - Query AWS S3 Tables Table Buckets using SQL"
+description: "Allows users to query AWS S3 Tables Table Buckets, providing information about the configuration, settings, and properties of your S3 table buckets."
+folder: "S3Tables"
+---
+
+# aws_s3tables_table_bucket
```

Comment:
Please follow the format mentioned in https://github.com/turbot/steampipe-plugin-aws/blob/main/docs/tables/aws_amplify_app.md#table-aws_amplify_app---query-aws-amplify-apps-using-sql